policyengine_us/parameters/gov/states/vt/tax/income/rates/alternative_minimum_tax/rate.yaml

```diff
@@ -0,0 +1,14 @@
+description: Vermont allows for an alternative minimum tax of this rate. 
+metadata:
+  label: Vermont alternative minimum tax rate
+  period: year
+  unit: /1
+  reference:
+    - title: Vermont §5822. Tax on income of individuals, estates, and trusts (a)(6)
+      href: https://legislature.vermont.gov/statutes/section/32/151/05822
+    - title: Vermont 2021 Income Tax Rate Schedules
+      href: https://tax.vermont.gov/sites/tax/files/documents/RateSched-2021.pdf#page=1
+    - title: Vermont 2022 Income Tax Return Booklet Forms and Instructions
+      href: https://tax.vermont.gov/sites/tax/files/documents/RateSched-2022.pdf#page=1
+values:
+  2021-01-01: 0.3
```

Comment:
should this be 3 or 30 %?

Reply:
You are right. It should be 3%. Changed.